distributed/tensor_parallelism/sequence_parallel_example.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+
+import torch
+import torch.multiprocessing as mp
+
+from torch.distributed._tensor import DeviceMesh
+from torch.distributed.tensor.parallel import parallelize_module, SequenceParallel
+from utils import cleanup, setup, ToyModel
+
+try:
+    from torch.distributed.tensor.parallel import (
+        SequenceParallel
+    )
+    SP_AVAILABLE = True
+except BaseException as e:
+    pass
```

Comment:
This is still needed for sequence parallel for now. After 2.1 release, we can remove it.